Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3104,4 +3110,43 @@ extension BrowserViewController {
             NavigationPath.handle(nav: path, with: self)
         }
     }
+    
+    func requestReviewIfNecessary() {
+        #if DEBUG
+        return
+        #endif
+        
+        let launchCount = Preferences.Review.launchCount.value
+        let threshold = Preferences.Review.threshold.value
+        
+        if threshold == 0 {
+            Preferences.Review.threshold.value = 14
```

Comment:
Is there a reason we don't just use 14 as the default threshold value when making the preference above?

Reply:
Fixed.